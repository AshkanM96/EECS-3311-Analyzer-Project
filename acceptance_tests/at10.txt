integer_constant(-0)
type_check
evaluate
reset
integer_constant(2)
type_check
evaluate
integer_constant(-4)
reset
type_check
evaluate
reset
boolean_constant(true)
evaluate
type_check
reset
boolean_constant(false)
evaluate
type_check
reset
