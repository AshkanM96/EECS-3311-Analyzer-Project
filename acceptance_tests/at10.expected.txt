  Expression currently specified: ?
  Report: Expression is initialized.
->integer_constant(0)
  Expression currently specified: 0
  Report: OK.
->type_check
  Expression currently specified: 0
  Report: 0 is type-correct.
->evaluate
  Expression currently specified: 0
  Report: 0
->reset
  Expression currently specified: ?
  Report: OK.
->integer_constant(2)
  Expression currently specified: 2
  Report: OK.
->type_check
  Expression currently specified: 2
  Report: 2 is type-correct.
->evaluate
  Expression currently specified: 2
  Report: 2
->integer_constant(-4)
  Expression currently specified: 2
  Report: Error (Expression is already fully specified).
->reset
  Expression currently specified: ?
  Report: OK.
->type_check
  Expression currently specified: ?
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: ?
  Report: Error (Expression is not yet fully specified).
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->boolean_constant(True)
  Expression currently specified: True
  Report: OK.
->evaluate
  Expression currently specified: True
  Report: True
->type_check
  Expression currently specified: True
  Report: True is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->boolean_constant(False)
  Expression currently specified: False
  Report: OK.
->evaluate
  Expression currently specified: False
  Report: False
->type_check
  Expression currently specified: False
  Report: False is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
